src/components/LoginButton.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
```

Comment:
Could you please add `Globo.com` license header in this file? You can find it in other files inside our project.